src/multi_auth/providers/facebook.cr

```diff
@@ -20,6 +20,8 @@ class MultiAuth::Provider::Facebook < MultiAuth::Provider
     user.last_name = fb_user.last_name
     user.location = fb_user.location
     user.description = fb_user.about
+    user.image = fb_user.picture[:data][:url]
```

Comment:
this code will raise an exception if fb_user.picture is nil, right? could we avoid it?